rclpy/test/test_node.py

```diff
@@ -1212,6 +1238,184 @@ def test_set_descriptor_read_only(self):
                 )
             )
 
+    def test_floating_point_range_descriptor(self):
+        # OK cases; non-floats are not affected by the range.
+        fp_range = FloatingPointRange(from_value=0.0, to_value=10.0, step=0.5)
+        parameters = [
+            ('from_value', 0.0, ParameterDescriptor(floating_point_range=[fp_range])),
+            ('to_value', 10.0, ParameterDescriptor(floating_point_range=[fp_range])),
+            ('in_range', 4.5, ParameterDescriptor(floating_point_range=[fp_range])),
+            ('str_value', 'I am no integer', ParameterDescriptor(floating_point_range=[fp_range])),
```

Comment:
I am not integer -> I am not float

Reply:
Thanks; addressed in https://github.com/ros2/rclpy/pull/357/commits/7381f62aeade16395c33dee943d7a5aa94173062.